### DISCUSSION TITLE
Merging two objects with duplicates

### DISCUSSION THREAD

**Discussion by kladrock** [open-policy-agent/community#393] [Policy Exchange]
Sharing the policy developed to merge two objects which had duplicate keys.

Input,

```
x = {"a":[true], "b":["foo"], "c":[4]}
y = {"b":["bar"], "d":["du"], "c":[100]}
```

expected output
```
{"a": [true],"b": ["bar","foo"],"c": [100,4],"d": ["du"]}
```

Policy shared at https://medium.com/@pushpalanka/hands-on-with-opa-2-merging-objects-de4f26d96baf
Playground link https://play.openpolicyagent.org/p/EUCS5ynfqt

**Comment by peteroneilljr** [comment]
Awesome thanks for sharing :)
